components/Contact/Contact.css

```diff
@@ -73,7 +90,7 @@
 
 @media (min-width: 48em) {
   .msrd .msrd-Contact-methods {
-    font: var(--font-07);
+    font: var(--font-06);
```

Comment:
Did you need to also update the letter spacing to `06`?

Reply:
Yes